### PR TITLE
Update Fluentd for 4.0

### PR DESCRIPTION
**Pull request by mburke5678**
Based on testing procedures

### CODE REVIEW

**Comment by mburke5678** [comment]
@jcantrill PTAL

**Comment by mburke5678** [comment]
@jcantrill PTAL

File: modules/efk-logging-fluentd-log-location.adoc [hunk 1]
```diff
@@ -5,19 +5,30 @@
...
+.Prerequisite
+
+Set cluster logging to the unmanaged state.
 
 .Procedure
 
 To set the output location for the Fluentd logs:
 
-. Edit the `LOGGING_FILE_PATH` parameter. You can specify a particular file or `STDOUT`:
+. Edit the `LOGGING_FILE_PATH`  parameter in the `fluentd` daemonset. You can specify a particular file or `STDOUT`:
```
Comment (jcantrill) [inline modules/efk-logging-fluentd-log-location.adoc]: I think its or 'console'

**Review by jcantrill — COMMENTED** [review]
nits... @richm  please additionally review

File: modules/efk-logging-fluentd-log-location.adoc (hunk 1 above)
Comment (mburke5678) [reply modules/efk-logging-fluentd-log-location.adoc]: @jcantrill You mean like in the example right below it? Ugh...

**Comment by mburke5678** [comment]
@anpingli Please take a look.

**Pull request merged**